realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/utils/HttpUtils.java

```diff
@@ -48,10 +52,10 @@ public static void startSyncServer() throws Exception {
 
         Headers responseHeaders = response.headers();
         for (int i = 0; i < responseHeaders.size(); i++) {
-            RealmLog.debug(responseHeaders.name(i) + ": " + responseHeaders.value(i));
+            Log.d(TAG, responseHeaders.name(i) + ": " + responseHeaders.value(i));
```

Comment:
@zaki50 because of this https://github.com/realm/realm-java/pull/4760/files/4e91142b6a9cdef94614bfb2c48651fd7d2434aa#diff-2365fb22852452691d8d2c0019e0ab5bR32